src/ImapLibrary2/__init__.py

```diff
@@ -486,3 +487,36 @@ def _get_all_emails(self):
         """Saves all existing emails to internal variable."""
         typ, mails = self._imap.uid('search', None, 'ALL')
         self._mails = mails[0].split()
+
+    def get_attachments_email (self, **kwargs) :
```

Comment:
There are methods named
get_matches_from_email
and
get_links_from_email

We should probably name this get_attachments_from_email to be consistent.

Reply:
Hi, it's done in my last commit